python/tk_multi_publish2/publish_tree_widget/publish_tree_widget.py

```diff
@@ -181,8 +181,8 @@ def build_tree(self):
             # TODO: because the context has changed for these items, the attached
             # tasks for the underlying item will be different. make sure the old
             # ones are replaced with the new
-            # self.__rebuild_tasks_r(item)
-            # _init_item_r(item)
+            self.__rebuild_tasks_r(item)
+            _init_item_r(item)
```

Comment:
This is already called as part of `self.__insert_item()`. You should only call it once.